test/e2e/__init__.py

```diff
@@ -51,3 +54,25 @@ def create_sagemaker_resource(
     )
 
     return reference, spec, resource
+
+
+def wait_sagemaker_endpoint_status(
```

Comment:
this should return `true` or `false` depending on whether it passed or not 

you can also return `actual_status` along with the Boolean

Reply:
lets change it when needed